docs/guide/deployment.md

```diff
@@ -52,7 +52,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           # choose node.js version to use
-          node-version: '14'
+          node-version: '16'
```

Comment:
No need to change the deployment guide. Use the lowest LTS version is OK here. (We should use 12 before :sweat_smile:)